### PR TITLE
Add version information for each command in doc

### PR DESCRIPTION
**Pull request by kay011**
About #1929 , I write a python script to generate each command's first support version. It will generate a csv file.  Finally the csv file is easy to convert to markdown format, but only do this, it is not completely correct. so I checked it manually. I found there are two mistake in the doc. and remarked them in the final markdown file. Committer can just copy the markdown file, and it is not necessary merge the code.

### CODE REVIEW

**Comment by PragmaTwice** [comment]
Thank you for your effort! Good to see it.

I think the original intension of #1929 is to add a new column into the website (https://kvrocks.apache.org/docs/supported-commands), rather than add a new file to the root directory of this project.

So technically, you can open pull requests to https://github.com/apache/kvrocks-website, instead of this repo.

File: cmd_support_version.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+| 213    | XREADGROUP           | -                 |                                                              |
+| 214    | XSETID               | v2.3.0            |                                                              |
+| 215    | BF.RESERVE           | v2.6.0            |                                                              |
+| 216    | BF.ADD               | v2.6.0            |                                                              |
+| 217    | BF.EXISTS            | v2.6.0            |                                                              |
+| 218    | BF.CARD              | v2.6.0            |                                                              |
+| 219    | BF.INFO              | v2.6.0            |                                                              |
+| 220    | BF.MADD              | v2.6.0            |                                                              |
+| 221    | BF.INSERT            | v2.7.0            |                                                              |
+| 222    | BF.MEXISTS           | v2.6.0            | **Doc mistake, the err command: bf.mexist**(manual check)    |
```
Comment (mapleFU) [inline cmd_support_version.md:225]: Would you mind also fixing the doc?

Comment (kay011) [reply cmd_support_version.md:225]: @mapleFU yeah, I will fix the doc in my pr!

**Comment by PragmaTwice** [comment]
Close since https://github.com/apache/kvrocks-website/pull/181 is merged. Thank you.

**Pull request closed without merging**